include/ofi_list.h

```diff
@@ -411,6 +411,7 @@ static inline void slist_insert_tail(struct slist_entry *item, struct slist *lis
 	else
 		list->tail->next = item;
 
+	item->next = NULL;
```

Comment:
Please add to the commit message that the slist iterators expect the item at the tail to have a null next pointer.  The slist_insert_head() function needs a similar change if the list is empty to start with.

Reply:
👍